apps/shell/shell.cc

```diff
@@ -88,13 +88,12 @@ void Shell::handleCommands()
     }
 
     auto builder = DefaultConfiguration::Builder {};
-    if (configFile.empty())
-        configFile = "./shell.conf";
-
-    try {
-        builder.load(configFile);
-    } catch (const std::exception& e) {
-        std::cout << "Can not load the config file: " << configFile << ", error: " << e.what();
+    if (!configFile.empty()) {
+        try {
+            builder.load(configFile);
+        } catch (const std::exception& e) {
+            std::cout << "Can not load the config file: " << configFile << ", error: " << e.what();
+        }
```

Comment:
Remove the default position settings with "./shell.conf" ? what's the reason we need to remove it.